### PR TITLE
feat: 446257 - Handle external auditing

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/config/storage.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const schema = Joi.object({
   connectionString: Joi.string().optional(),
   account: Joi.string().required(),
   eventTable: Joi.string().default('events'),
+  externalEventTable: Joi.string().default('external'),
```
Comment (jbarnsley10) [inline app/config/storage.js:8]: Can we have the name 'external_events' please

File: app/inbound/handle-event/external-event.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { createRow } = require('./create-row')
+const { EXTERNAL_EVENT } = require('../../constants/event-types')
+const { getClient } = require('../../storage')
+const { createIfNotExists } = require('./create-if-not-exists')
+const saveExternalEvent = async (event) => {
+  const entity = createRow(event.partitionKey ?? event.id, event.id, EXTERNAL_EVENT, event)
```
Comment (jbarnsley10) [inline app/inbound/handle-event/external-event.js:6]: We will need to enhance this to save in mutliple ways depending on the event type

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
Looks good. Just default table value name change, and we can enhance the 'save' method later
Have you checked if we need a Managed Identity permissions FFC PR for reading/writring/creating the new table?

File: app/inbound/handle-event/external-event.js (hunk 2 above)
Comment (whitewaterdesign) [reply app/inbound/handle-event/external-event.js:6]: have added more coverage for this

**Review by jbarnsley10 — APPROVED** [review]
LGTM

**Pull request merged**